src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfirm.tsx

```diff
@@ -98,22 +173,22 @@ export const LimitOrderConfirm = () => {
               </Row.Label>
               <Row.Value textAlign='right'>
                 <HStack>
-                  <SwapperIcon swapperName={SwapperName.Zrx} />
-                  <RawText>0x</RawText>
+                  <SwapperIcon swapperName={SwapperName.CowSwap} />
+                  <RawText>{SwapperName.CowSwap}</RawText>
                 </HStack>
               </Row.Value>
             </Row>
             <Row px={2}>
               <Row.Label>
                 <Text translation='limitOrder.expiration' />
               </Row.Label>
-              <TransactionDate blockTime={Date.now() / 1000} />
+              <TransactionDate blockTime={quoteExpirationTimestamp ?? 0} />
```

Comment:
q: do we even want to display the tx time if this var is undefined? It would end up showing some weird time

Reply:
You're right, we dont. In this case it's more to make typescript happy, because the selector only returns undefined if there is no active quote - which isnt possible here as they need a quote to get to this page